examples/server/index.html

```diff
@@ -30,15 +30,15 @@
 
 <h2>Options</h2>
 <div class="option">
-    <input id="use-datachannel" checked="checked" type="checkbox"/>
```

Comment:
Revert this, it's unrelated to H.264

Reply:
Sorry about that. I changed to debug faster and I forgot to revert before committing

Reply:
No need to apologize, I perfectly understand why you had it in your local copy, just pointing out the commit looked like an oversight :)